src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +91,55 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={titleSearchQuery}
+              onChange={handleSearchFieldChange}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {errorMessage && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames(
+                'button is-light',
+                { 'is-loading': loading },
+              )}
+              onClick={() => createMovie()}
```

Comment:
```suggestion
              onClick={createMovie}
```

Reply:
It won't work because the function doesn't include the 'event' parameter